football-manager/src/lib/Pages/Club/clubNavBar.svelte

```diff
@@ -0,0 +1,14 @@
+
+<script>
+  import NavButton from "../../Components/NavButton.svelte";
+  export let id;
+</script>
+
+<div class="bg-slate-400 dark:bg-slate-800 w-full rounded-xl p-2 flex gap-2">
+  <NavButton buttonText="Home" location="/team/{id}" {id} />
+  <NavButton buttonText="Squad" location="/team/{id}/squad" {id} />
```

Comment:
Should probably teams? Since a team contains a squad.

Reply:
I copied the part you told me to copy. You also use team

Reply:
Tänkte mer att på club sidan ska du hitta lagen, och på team sidan ska du hitta spelarna.

Reply:
Menar du att home ska vara club?

Reply:
Nej. Blir väll Home -> Club -> Team.

Reply:
Jag tänker att när man trycker på home så kommer man till club sidan, tänker jag fel?

Reply:
location på den blir "/club/{id}"

Reply:
Där tänker du rätt, jag syftar mer på squads knappen. Den borde vara teams.

Reply:
aha titeln på button borde vara teams, för när man klickar på den så kommer man till lagen som är uppdelade i pojkar, flickor och seniorer

Reply:
Japp, plus någon annan route. 
Typ '/club/id/teams' eller liknande.

Reply:
Löser det